primitives/weights/src/weight_v2.rs

```diff
@@ -160,6 +160,26 @@ impl Weight {
 		Self { ref_time: 0 }
 	}
 
+	/// Constant version of Add with u64. Is only overflow safe when evaluated at compile-time.
+	pub const fn add(self, scalar: u64) -> Self {
+		Self { ref_time: self.ref_time + scalar }
+	}
+
+	/// Constant version of Sub with u64. Is only overflow safe when evaluated at compile-time.
+	pub const fn sub(self, scalar: u64) -> Self {
+		Self { ref_time: self.ref_time - scalar }
+	}
+
+	/// Constant version of Div with u64. Is only overflow safe when evaluated at compile-time.
+	pub const fn div(self, scalar: u64) -> Self {
+		Self { ref_time: self.ref_time / scalar }
+	}
+
+	/// Constant version of Mul with u64. Is only overflow safe when evaluated at compile-time.
```

Comment:
```suggestion
	/// Constant version of Sub with u64.
	///
	/// Is only overflow safe when evaluated at compile-time.
	pub const fn sub(self, scalar: u64) -> Self {
		Self { ref_time: self.ref_time - scalar }
	}

	/// Constant version of Div with u64.
	///
	/// Is only overflow safe when evaluated at compile-time.
	pub const fn div(self, scalar: u64) -> Self {
		Self { ref_time: self.ref_time / scalar }
	}

	/// Constant version of Mul with u64.
	///
	/// Is only overflow safe when evaluated at compile-time.
```

Reply:
@bkchr you want the docs changed? ^^

Reply:
Yes, some nitpick. I would just like to have them on their own line.

Reply:
Will do next week.

Reply:
You can click on the "commit suggestion" button in github.

Reply:
Not sure why, but these on github commit are fishy to me. ^^